bin/unienv

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env node
```

Comment:
Can you explain why this file should exist?

Reply:
Thanks, It was leftover from an attempt to rebase on upstream.  I've removed it now.